src/Valit/ValitRulePropertyExtensions.cs

```diff
@@ -44,14 +44,27 @@ public static class ValitRulePropertyExtensions
             return rule;
         }
 
+        public static IValitRule<TObject, TProperty> WithMessage<TObject, TProperty>(
+            this IValitRule<TObject, TProperty> rule, Func<string> message) where TObject : class
```

Comment:
can you add "Func" sufix, just for consistency ?

Reply:
It has been done :-)